src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -220,6 +220,7 @@ def load_command_table(self, _):
                               table_transformer=aks_show_nodepool_snapshot_table_format)
         g.custom_command('create', 'aks_nodepool_snapshot_create',
                          supports_no_wait=True)
+        g.custom_command('update', 'aks_nodepool_snapshot_update')
```

Comment:
add 'supports_no_wait=True' to be consistent with other nodepool snapshot commands.

Reply:
fixed